framework/tlvf/tlvf.py

```diff
@@ -779,7 +779,7 @@ def addClassParam(self, obj_meta, param_name, param_type, param_type_info, param
                     lines_h.append( "}" )
                 else:
                     # add default value to init func
-                    lines_cpp.append("m_%s = (%s*)m_%s__;" % ( param_name, param_type, self.MEMBER_BUFF_PTR))
+                    lines_cpp.append("m_%s = reinterpret_cast<%s*>(m_%s__);" % ( param_name, param_type, self.MEMBER_BUFF_PTR))
```

Comment:
Please squash this in the other commit doing the same thing somewhere else.

Reply:
Done